subsys/profiler/Kconfig

```diff
@@ -9,10 +9,13 @@ menuconfig PROFILER
 
 if PROFILER
 
-config PROFILER_MAX_NUMBER_OF_CUSTOM_EVENTS
-	int "Maximum number of stored custom event types"
+config PROFILER_MAX_NUMBER_OF_EVENTS
+	int "Maximum number of stored event types"
 	default 32
-	range 0 32
+	range 0 255
+	help
+	  Maximum number of stored event types.
+	  Note that some subsystems can have their own event types which count towards the maximum number of events.
```

Comment:
>Note that some subsystems can have their own event types which count towards the maximum number of events.
The above could be simplified to `set to high enough` or `value depends on what is using profiler`. Both obvious and meaningless.

I don't think this comment belong here. If somebody enables profiling of event manager, the documentation of the symbol should tell how many profiler events are needed.

Reply:
Added information under CONFIG_EVENT_MANAGER_PROFILER_ENABLED.
Why do not leave it in form as it is under PROFILER_MAX_NUMBER_OF_EVENTS?

Reply:
Ok, I removed it from here